scheduler/scd_utils.py

```diff
@@ -249,7 +254,7 @@ def get_relevant_lines(self, yyyymmdd, hhmm):
     scd_util.add_line("20190414", "10:43", "twofsound")
     scd_util.add_line("20190414", "10:43", "twofsound", prio=2)
     scd_util.add_line("20190414", "10:43", "twofsound", prio=1, duration=89)
-    scd_util.add_line("20190414", "10:43", "twofsound", prio=1, duration=24)
+    #scd_util.add_line("20190414", "10:43", "twofsound", prio=1, duration=24)
```

Comment:
maybe move the main in this file to a separate test file? I think it'd be good to have a test code example somewhere, but I don't think this is a good place

Reply:
I moved this to a new file.